app/modules/front-door/locals.tf

```diff
@@ -1,8 +1,11 @@
 locals {
+  service_name = "back-office-applications-docs"
+  environment  = var.environment
```

Comment:
Do we need a copy of environment as local?

Reply:
removed from locals on new commit